index.css

```diff
@@ -1,103 +1,240 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300,400,500,600&display=swap');
+
 * {
-  margin: 0;
-  padding: 0;
   box-sizing: border-box;
 }
 
 body {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  width: 100vw;
-  font-family: 'Arial', sans-serif;
+  font-family: c, sans-serif;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  max-width: 1200px;
+  height: 100vh;
+  margin: auto;
+  padding: 0;
+  background-image: url(./book-bg.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
```

Comment:
Kindly reduce the fontsize for your project so it looks similar to the design given 👍